great_expectations/data_context/store/ge_cloud_store_backend.py

```diff
@@ -188,6 +187,12 @@ def __init__(
         if not self._suppress_store_backend_id:
             _ = self.store_backend_id
 
+        if http_handler is None:
```

Comment:
I don't see where we are passing this in as a parameter.

For now, please consider removing http_handler as a parameter from __init__ and if/when we do add it back, it should be a required dependency